container/root/run.d/08-php-opcache.sh

```diff
@@ -1,9 +1,8 @@
-#!/usr/bin/with-contenv bash
+#!/bin/bash
```

Comment:
Just a note that 755 -> 644 file mode change will require the file to be sourced, or executed indirectly via /bin/bash - running the script directly (./08-php-opcache.sh for example) will result in a permission denied.  If the file is being read this way, then the perm change is ok.